src/main/java/it/sijmen/movienotifier/service/cinemas/pathe/PatheShowing.java

```diff
@@ -217,4 +221,46 @@ public boolean equals(Object o) {
     public int hashCode() {
         return (int) (getId() ^ (getId() >>> 32));
     }
-}
+
+    private static final SimpleDateFormat format1 = new SimpleDateFormat("EEE d MMMM HH:mm");
+    private static final SimpleDateFormat format2 = new SimpleDateFormat("HH:mm");
+
+    public String toString(PatheShowing showing) {
```

Comment:
Perhaps we could call this method `toMessageString`? For a `toString` I'd expect a default (generated) toString with all properties, no special formatting.

Reply:
good idea!